pkg/ebpftracer/tracer.go

```diff
@@ -558,6 +559,7 @@ func (t *Tracer) removeCgroups(cgroupIDs []cgroup.ID) {
 	t.removedCgroupsMu.Lock()
 	for _, id := range cgroupIDs {
 		delete(t.cgroupEventPolicy, id)
+		metrics.AgentTracerCgroupEventPolicies.Dec()
```

Comment:
I am not sure if this will be a good metric that way tbh. It would be better to separate them into two separate.

Reply:
This is temp metric to understand if we are loosing cleanups. Profiling shows allocations for dns lru cache which is added for each cgroup policy.